src/components/Select.tsx

```diff
@@ -0,0 +1,20 @@
+export type SelectProps = {
+  className?: string;
+  id: string;
+  name: string;
+  options: { key: number | string; label: string; value: number | string }[];
+};
+
+const Select = ({ className = '', id, name, options }: SelectProps) => {
```

Comment:
pitäsköhä toi inputClassName vaa siirtää johki mistä se exportattais, ja sit käytettäis näis defaulttina eikä otettais proppina mitää classnamesia?

Reply:
Pitäis ehkä tehdä tän kanssa?

https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply

Reply:
mä sanoisin et ei käytetä applya:

"If you’re going to use @apply, use it for very small, highly reusable things like buttons and form controls — and even then only if you’re not using a framework like React where a component would be a better choice."